app/main.py

```diff
@@ -83,6 +83,7 @@ def ftp_params(services):
             "password": os.getenv("SEFT_FTP_PASS", "password"),
             "host": os.getenv("SEFT_FTP_HOST", "127.0.0.1"),
             "port": int(os.getenv("SEFT_FTP_PORT", 2121)),
+            "working_directory": os.getenv("SEFT_PUBLISHER_FTP_FOLDER", "/")
```

Comment:
Could we please have a unit test to show that if SEFT_PUBLISHER_FTP_FOLDER has multiple separators, or ends in a separator, that we handle it correctly? Probably requires a little static method to do the read from the env.

Example of a similar test:
https://github.com/ONSdigital/sdx-common/blob/master/sdx/common/test/test_transformer.py#L464